src/doc/rustc/src/platform-support.md

```diff
@@ -202,6 +202,7 @@ target | std | host | notes
 `powerpc-wrs-vxworks` | ? |  |
 `powerpc-wrs-vxworks-spe` | ? |  |
 `powerpc64-unknown-freebsd` | ✓ | ✓ | PPC64 FreeBSD (ELFv1 and ELFv2)
+`powerpc64le-unknown-freebsd` | ✓ | ✓ | PPC64LE FreeBSD
```

Comment:
std and host toolchains requires adjustments to the dockerfiles and dist infrastructure found in this repository. These checkmarks should be either removed or the CI adjusted.

Reply:
Where should those adjustments be added? I looked into https://github.com/rust-lang/rust/tree/master/src/ci/docker/host-x86_64, but I don't see a folder with already supported powerpc64-unknown-freebsd.

Reply:
I don't think we distribute prebuilt artifacts for `powerpc64-unknown-freebsd` either, which would make the annotation for powerpc64 above incorrect as well.

Reply:
OK, I'll later modify https://github.com/rust-lang/rust/blob/master/src/ci/docker/scripts/freebsd-toolchain.sh and create  dist-powerpc64le-freebsd directory. powerpc64 should probably have those annotations removed, since it currently requires non-upstreamable patches.

Reply:
One issue about freebsd-toolchain.sh - currently, an out-of-tree binutils patch is necessary to build and use binutils. Will that be acceptable?

Reply:
@nagisa waiting for you to reply to this. Thanks

Reply:
I think these changes should be pursued as a separate PR (and these checkmarks removed for the time being).

I'm not super familiar with what's appropriate or not for the bootstrap stuff.